edge/test/integration/utils/helpers/helpers.go

```diff
@@ -319,11 +312,6 @@ func HandleAddAndDeletePods(operation string, edgedpoint string, UID string, con
 		}
 		req, err = http.NewRequest(http.MethodDelete, edgedpoint, bytes.NewBuffer(respbytes))
 	}
-	if err != nil {
```

Comment:
how about move this err into switch,

Reply:
I read it again and found that it's actually a bug. The returned err from http.NewRequest is not visible outside the switch statement. There are also some duplication in code. I will refactor it.

Reply:
yes, can write a func here, also same with line 273

Reply:
Done with [a new separate commit](Done with a new separate commit https://github.com/kubeedge/kubeedge/pull/1009/commits/6c685fcbd855c240131ae35e89e1b0c464edbb08). PTAL

Reply:
Great fix!